modal/cli/launch.py

```diff
@@ -17,16 +24,25 @@
 )
 
 
-def _launch_program(name: str, args) -> None:
-    contents = (Path(__file__).parent / "programs" / name).read_text()
-    contents = contents.replace("args: Dict[str, Any] = {}", f"args: Any = {repr(args)}")
+def _launch_program(name: str, filename: str, args: Dict[str, Any]) -> None:
+    os.environ["MODAL_LAUNCH_LOCAL_ARGS"] = json.dumps(args)
```

Comment:
maybe i'm missing something but why can't you just pass this to `func` directly? i.e. invoke `func(args)` instead